camel/agents/task_agent.py

```diff
@@ -39,6 +39,8 @@ class TaskSpecifyAgent(ChatAgent):
             the task. (default: :obj:`None`)
         word_limit (int): The word limit for the task prompt.
             (default: :obj:`50`)
+        output_language (str): The language to be used in the specification
+            (default: :obj:`English`)
```

Comment:
Let's make the default as `None`. Only `None` we do nothing, otherwise we always call `reset_output_language` in `ChatAgent`. This is because the base model or prompt could be in a different language. So by default, we do nothing. But `English` would not be always the default language.